db/schema.rb

```diff
@@ -88,6 +88,19 @@
   add_index "bundle_products", ["bundle_product_id"], name: "fk_bundle_prod_prod", using: :btree
   add_index "bundle_products", ["product_id"], name: "fk_bundle_prod_bundle", using: :btree
 
+  create_table "card_readers", force: :cascade do |t|
+    t.integer  "control_device_id", limit: 4, null: false
+    t.boolean  "entrance",                    null: false
```

Comment:
haha yeah I actually could have just left it out for now, after like 30s of trying to think about what a good name was I just moved on